### ISSUE TITLE
Glossary of PM Tools 

### ISSUE THREAD

**Issue by kcoronel** [hackforla/product-management#259]
### Overview
In an effort to help support a learning experience for our Community of Practice. We will create a list of PM tools available in the market and take account of which members of our community can offer trainings or support for novices. We will also create a glossary of the tools on our list. This can be useful for folks inside and outside of our community. 

### Action Items
Different types of product management software - MSProjects, Monday.com, JIRA, MS Dynamics, Smartsheets, Trello, Aha!, Product Plan, Asana
Analytics: FullStory, Heap, Mixpanel, Tableau, Snowflake, AppCues, Pendo 

### Resources/Instructions
#94 
[Draft Glossary of PM Tools](https://docs.google.com/spreadsheets/d/1Co_QhlpT5b2GXTIdbgzNxoFe0EVTCL0cczs2X5PUWF0/edit#gid=0)

**Comment by ExperimentsInHonesty** [comment]
@kcoronel it looks like you are assigned to this issue.  Do you want to keep working on it, or would you like to be unassigned?

**Comment by kcoronel** [comment]
@ExperimentsInHonesty I would like to keep it. I am down to two contracts instead of three and 1/2 so I will have time to complete a first draft in time for Wednesday's PM meeting.

**Comment by kcoronel** [comment]
I started a google sheet with glossary terms, it is linked above in the overview.

**Comment by ExperimentsInHonesty** [comment]
@kcoronel its looking good.  I added some formatting to it, to make it easier to read.  What are the next steps?

**Comment by ExperimentsInHonesty** [comment]
@kcoronel its looking good. I added some formatting to it, to make it easier to read. What are the next steps?

**Comment by kcoronel** [comment]
@ExperimentsInHonesty I would recommend the next steps bringing this up to the entire PM community for input that I may have missed.

**Comment by ExperimentsInHonesty** [comment]
@priyatalwar lets add this to our roadmap for future PM Communities of practice.

**Comment by ExperimentsInHonesty** [comment]
Posted the following message on the PM CoP

> Hi All. Hope you  are having a great weekend.  Please check out this issue that @Karen (she/her/ella) (PST) is working on.  It's a Glossary of PM Tools.  Please add any  PM Tools your team uses or knows about. https://github.com/hackforla/product-management/issues/259

**Comment by ExperimentsInHonesty** [comment]
We might need an issue that is a guide to how to make a tool glossary and competitive analysis, using our spreadsheet as the example.

**Comment by edwardsarah** [comment]
Similar to https://github.com/hackforla/product-management/issues/314. We may need to eliminate one or both of these issues.

**Comment by kcoronel** [comment]
@edwardsarah agreed, I think this issue could be closed as - "repetitive  to issue #314.
